binrz/man/rz-ax.1

```diff
@@ -1,100 +1,100 @@
-.Dd Dec 28, 2020
+.Dd Jan 24, 2024
 .Dt RZ_AX 1
 .Sh NAME
 .Nm rz-ax
 .Nd rizin base converter
 .Sh SYNOPSIS
 .Nm rz-ax
-.Op Fl ebBsSvxkKh
+.Op Fl abBdefFhiIkLlNnorsStuvwxC
 .Op [expr] ...
 .Sh DESCRIPTION
 This command is part of the rizin project.
 .Pp
-This command allows you to convert values between positive and negative integer, float, octal, binary and hexadecimal values.
+This command allows you to convert values between positive and negative integers, floats, octal, binary, ternary, and hexadecimal values.
 .Sh OPTIONS
 .Bl -tag -width Fl
+.It Fl l
+Append newline to output (for -E/-D/-r/..)
+.It Fl a
+Show ASCII table
 .It Fl b
-Convert from binary string to character (rz-ax \-b 01000101)
-.It Fl k
-Keep the same base as the input data
+Convert from binary string to character (rz-ax -b 01000101 01110110)
+.It Fl B
+Convert from string to binary (rz-ax -B hello)
+.It Fl d
+Force output as integer (rz-ax -d 3 outputs 3 instead of 0x3)
 .It Fl e
 Swap endian.
+.It Fl f
+Floating point arithmetic (rz-ax -f 6.3+2.1)
 .It Fl F
-Read C strings from stdin and output in hexpairs. Useful to load shellcodes
+Read C strings from stdin and output in hex pairs. Useful to load shellcodes
+.It Fl h
+Show usage help message
 .It Fl i
-Dump stdin to C array in stdout (xxd replacement)
+Dump as C byte array
 .It Fl I
 Convert LONG to/from IP ADDRESS
-.It Fl l
-Append newline to the decoded output for human friendly-ness
-.It Fl K
-Show randomart key asciiart for values or hexpairs
-.It Fl s
-Convert from hex string to character (rz-ax \-s 43 4a 50)
-.It Fl S
-Convert from character to hex string (rz-ax \-S C J P)
+.It Fl k
+Keep the same base as the input data
+.It Fl L
+Convert binary to hexadecimal (rz-ax -L 111111111 outputs 0x1ff)
 .It Fl n
-Show hexpairs from integer value
+Show binary number from integer value
```

Comment:
this is wrong
```
rz-ax -n 0x11223344
44332211
```
this is hexpairs, not a binary number.

Reply:
changing it in the `rz-ax -h` also